x/dex/client/cli/tx/tx_update_quantity_tick_size.go

```diff
@@ -0,0 +1,50 @@
+package tx
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	cutils "github.com/sei-protocol/sei-chain/x/dex/client/utils"
+	"github.com/sei-protocol/sei-chain/x/dex/types"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdUpdateQuantityTickSize() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-quantity-tick-size [update-quantity-tick-size-file]",
+		Short: "Update quantity tick size for a market",
```

Comment:
For my info, what's the difference between this and the price?

Reply:
Price is the price at which the order is placed, the quantity is what amount of the asset the order is for. We want to add tick size to both instead of only price as was the case previously